package.json

```diff
@@ -17,12 +17,13 @@
   },
   "homepage": "https://github.com/dontcallmedom/canon-singing#readme",
   "dependencies": {
-    "audio-recorder-polyfill": "^0.3.6",
+    "audio-recorder-polyfill": "^0.3.8",
     "body-parser": "^1.19.0",
     "express": "^4.17.1",
     "express-fileupload": "^1.2.0",
     "file-saver": "^2.0.2",
     "fluent-ffmpeg": "^2.1.2",
     "nanoid": "^3.1.12"
-  }
+  },
+  "devDependencies": {}
```

Comment:
why?

Reply:
No reason, I probably issued an npm install command that added this automatically, will drop